protos/bottle/fulfillment/v1/tax_line.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+message TaxLine {
+  string id = 1;
+  string name = 2;
+  int64 price = 3;
```

Comment:
Just a thought but `amount`  may be a more appropriate term for this value. You don't typically refer to taxes in terms of "price".

Reply:
I'll update.

Reply:
Updated to `amount`.